app/models/tag.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../lib/acts_as_taggable_on/tag.rb"
```

Comment:
(we don't need this as `app/lib` is preloaded, tested without it and it works)

Reply:
Really? If I remove this line my tests, `spec/lib/acts_as_taggable_on/tag_spec.rb` fail and when I try adding pry to the `after_commit` block I find that it never gets there. I assumed it had to do with load order and found that requiring it here fixed the problem and ensured that callback was triggered

Reply:
Rails is too magic :D